index.html

```diff
@@ -106,7 +107,7 @@ <h2 class="stories1"> Website Protfolio </h2>
                 </ul>
               </div>
           </div>            
-        </div>
+       </div>
 
 
         <div class="box2"> 
```

Comment:
- [ ] There are overlapping of texts and images here. Kindly look into them and fix them. Please, take a look.

### Your Design
![your-second-work](https://user-images.githubusercontent.com/26597432/206869815-08ecf482-e25b-4842-9032-21f67ee81e3e.png)

### Figma Design
![figma-second-work](https://user-images.githubusercontent.com/26597432/206869828-00f644f3-8166-4c1e-ac0e-0b15c82aa0b8.png)